### PR TITLE
Device inits corrected

### PR DESCRIPTION
**Pull request by playfulgod**
corrected inits

### CODE REVIEW

File: devices.js [hunk 1]
```diff
@@ -2203,8 +2203,9 @@
 
       ],
       "lunch": "cm_vs920-userdebug",
-      "init": "init.vs920.rc",
+      "init": "init.target.rc",
       "name": "LGE VS920",
+      "inits":["init.target.rc","init.iproj.rc","init.i_vzw.rc","init.iprj.rc"]
```
Comment (koush) [inline devices.js]: missing comma

File: devices.js [hunk 2]
```diff
@@ -2226,7 +2227,8 @@
 
       ],
       "lunch": "cm_vs910-userdebug",
-      "init": "init.vs910.rc",
+      "init": "init.qcom.rc",
+      "inits":["init.qcom.rc","init.bryce.rc"]
       "name": "LGE VS910",
```
Comment (koush) [inline devices.js]: missing trailing comma

``` javascript
"inits":["init.qcom.rc","init.bryce.rc"],
```

File: devices.js [hunk 3]
```diff
@@ -2238,7 +2240,8 @@
 
       ],
       "lunch": "cm_ms910-userdebug",
-      "init": "init.ms910.rc",
+      "init": "init.qcom.rc",
+      "inits":["init.qcom.rc","init.bryce.rc"]
```
Comment (koush) [inline devices.js]: here too

**Comment by playfulgod** [comment]
Should be good to go now.

**Pull request merged**